metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -18,6 +19,13 @@ import org.eclipse.lsp4j.ShowMessageRequestParams
  */
 object Messages {
 
+  def showError(msg: String): MessageParams = {
```

Comment:
```suggestion
  def errorMessageParams(msg: String): MessageParams = {
```
? Since the method does not show anything and the name would indicate a side effect happening.

Reply:
ahhhh I get what you were saying before. Sorry I misunderstood at first. I'll change this.